sxw_soilwat.c

```diff
@@ -94,7 +94,10 @@ void _sxw_sw_setup (RealF sizes[]) {
     }
   }
 
-	SW_VPD_init();
+  /* We no longer need to call `SW_VPD_init` since SOILWAT2 v3.5.0. SOILWAT2
+     calls `SW_VPD_init` at the beginning of each year (for dynamic CO2-effects)
+  */
+  // SW_VPD_init();
 }
```

Comment:
@dschlaep I believe we can remove the above commented out code before merging in feature_space?